pkg/cli/cmd/resource/list/list_test.go

```diff
@@ -61,6 +61,24 @@ func Test_Validate(t *testing.T) {
 				Config:         configWithWorkspace,
 			},
 		},
+		{
+			Name:          "Valid List Command with environment",
+			Input:         []string{"Applications.Core/containers", "-e", "test-env"},
```

Comment:
Should we check `--environment` just in case?

Reply:
@ytimocin - I've added a couple of test cases to include the full `--environment` flag 😃